src/style.css

```diff
@@ -1 +1,71 @@
-/* add styles here */
+body {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```

Comment:
Remember to use fallback fonts, i.e., you should provide alternative font-families in case the main one doesn't load. In your case, 'Avenir' might not load on all systems. The fallback fonts should be similar in style to your main font and should be ones that are widely available. For example, you could use 'Arial, sans-serif' as fallbacks. (Checklist: Remember to use fallback fonts - alternative font-family in case the main one doesn't work)